src/models/mnist_module.py

```diff
@@ -122,8 +123,17 @@ def configure_optimizers(self):
         Examples:
             https://pytorch-lightning.readthedocs.io/en/latest/common/lightning_module.html#configure-optimizers
         """
+        optimizer = self.hparams.optimizer(params=self.parameters())
+        scheduler = self.hparams.scheduler(optimizer=optimizer)
+
         return {
-            "optimizer": self.hparams.optimizer(params=self.parameters()),
+            "optimizer": optimizer,
+            "lr_scheduler": {
+                "scheduler": scheduler,
+                "monitor": "val/acc_best",
```

Comment:
I think schedulers are usually optimized on validation loss.

Accuracy can remain the same for a longer time even though loss is improving, so accuracy breakthrough might get delayed